### DISCUSSION TITLE
建议加入cuda是否可执行的判断

### DISCUSSION THREAD

**Discussion by qxk2005** [jianchang512/pyvideotrans#90] [General]
使用的编译后开箱即用版本，但是cuda一直没有生效过。本地是N卡，也部署了CUDA，可以跑whisperdesktop版本的cuda加速。不知道为什么pyvideotrans不行，建议加入一个cuda判断的按钮，给出是否可以跑的建议，方便使用，谢谢了。
![14517FFC](https://github.com/jianchang512/pyvideotrans/assets/61008180/60f6ee64-8401-4828-8a9f-2b5cb76efcbc)

**Comment by jianchang512** [comment]
cuda需要使用源码自行部署，预编译版不支持。
readme页面有说明
具体启用方法
https://github.com/jianchang512/pyvideotrans#cuda-%E5%8A%A0%E9%80%9F%E6%94%AF%E6%8C%81
